build.gradle

```diff
@@ -465,15 +465,16 @@ project(':iceberg-aws') {
     implementation libs.jackson.databind
     implementation libs.jackson.core
 
-    compileOnly libs.awssdk.url.connection.client
-    compileOnly libs.awssdk.apache.client
-    compileOnly libs.awssdk.auth
-    compileOnly libs.awssdk.s3
-    compileOnly libs.awssdk.kms
-    compileOnly libs.awssdk.glue
-    compileOnly libs.awssdk.sts
-    compileOnly libs.awssdk.dynamodb
-    compileOnly libs.awssdk.lakeformation
+    implementation(platform(libs.awssdk.bom))
```

Comment:
I think, it's better to keep the original configuration `compileOnly` here.

Reply:
changed it by mistake, thanks!
switched back to compileOnly